stock_move_value/models/stock_move.py

```diff
@@ -60,7 +60,7 @@ def _action_done(self, cancel_backorder=False):
             }
         moves = super()._action_done(cancel_backorder)
         for move in moves:
-            move.move_value = sum(move.stock_valuation_layer_ids.mapped("value"))
+            move.move_value = sum(move.sudo().stock_valuation_layer_ids.mapped("value"))
```

Comment:
Can we change `for move in moves:` above to `for move in moves.sudo():` instead?

Reply:
Same as above. Please disregard my suggestion.